src/data/set/pointwise/smul.lean

```diff
@@ -378,6 +378,15 @@ end vsub
 
 open_locale pointwise
 
+@[to_additive] lemma smul_image [has_smul α β] [has_smul α γ] (f : β → γ) (s : set β) (a : α) :
+  (∀ b, a • f b = f (a • b)) → a • f '' s = f '' (a • s) :=
+image_comm
```

Comment:
As above, lets flip the equalities (and perhaps the name too?)

Reply:
If I flip the name, it will be a third lemma that wants to be called `image_smul` 😦

Reply:
Let's at least flip the statement since you did that for finset.

I'd argue that the existing `image_smul` lemma should be called `smul_image` anyway...